app/addons/permissions/components/PermissionsItem.js

```diff
@@ -17,15 +17,9 @@ import React from 'react';
 const PermissionsItem = ({removeItem, section, type, value}) => {
 
   return (
-    <li className="permissions__entry">
+    <li>
```

Comment:
There's a couple of tests that rely on this CSS class. It's easier to update them now.
The tests use something like `.permissions__admins .permissions__entry` so it should be enough to replace them with `.permissions__entry li`.